packages/app/components/Today/OpenQueueCard.tsx

```diff
@@ -119,71 +127,55 @@ const OpenQueueCard = ({
 
   return (
     <PaddedCard
-      headStyle={{ background: "#F3F5F7" }}
+      headStyle={{ background: "#25426C", color: "#FFFFFF" }}
       className={"open-queue-card"}
-      title={
-        staffList.map((staffMember) => staffMember.name).join(", ") ||
-        "No Staff Checked In!"
-      }
-      extra={queue.startTime && queue.endTime && formatQueueTime(queue)}
+      title={`${queue.room} ${
+        queue.isProfessorQueue ? `(Professor queue)` : ``
+      }`}
+      extra={`${queue.queueSize} in queue`}
     >
       <QueueInfoRow>
-        <HeaderDiv>{queue.room}</HeaderDiv>
+        <HeaderDiv>
+          <QuestionNumberSpan>{queue.staffList.length}</QuestionNumberSpan>{" "}
+          staff checked in
+        </HeaderDiv>
         <RightQueueInfoRow>
-          {!queue.allowQuestions && (
-            <Tooltip title="This queue is no longer accepting questions">
-              <StopOutlined
-                style={{ color: "red", fontSize: "24px", marginRight: "8px" }}
-              />
-            </Tooltip>
-          )}
-          <QueueSizeColorDiv>
-            <QuestionNumberSpan>{queue.queueSize}</QuestionNumberSpan> in queue
-          </QueueSizeColorDiv>
+          <Space
+            direction="vertical"
+            size="middle"
+            style={{ display: "flex", alignItems: "center" }}
+          >
+            {!queue.allowQuestions && (
```

Comment:
The spacing looks a bit weird when this "not accepting questions" tag component shows up - I wonder if it would look better inline next to the `# staff checked in` text? Like just having it directly to the right of it

Reply:
![image](https://user-images.githubusercontent.com/11274412/162634766-ad3f9161-929f-4401-ae73-00ca684dacff.png)
not sure if im a fan

Reply:
Hmm I feel like that's still better than moving the `Open Queue` button down, which looks a bit awkward to me 🤔

Reply:
Or alternatively I think if you kept it as is (above the button) but right aligned everything, I think that would look decent as well